components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -205,8 +205,20 @@ tweaking their $case/namelist_scream.xml file.
 
     <!-- Radiation -->
     <rrtmgp inherit="physics_proc_base">
-      <active_gases>h2o, co2, o3, n2o, co, ch4, o2, n2</active_gases>
       <column_chunk_size>1280</column_chunk_size>
+      <!-- Radiatively active gases; surface values set to F2010 settings taken from EAM  -->
+      <!-- Note that h2o concentrations are just taken from qv, o3 is prescribed for now, -->
+      <!-- o2 is hard-coded as a constant, CFCs are ignored                               -->
+      <active_gases>h2o, co2, o3, n2o, co, ch4, o2, n2</active_gases>
+      <ch4vmr      > 1807.851e-9  </ch4vmr>
+      <co2vmr      > 388.717e-6   </co2vmr>
+      <!-- If co2vmr_rad is set and greater than 0, it will override any values of co2vmr for rad calcs -->
+      <co2vmr_rad  > -1.0         </co2vmr_rad>
```

Comment:
So, if co2vm_rad>0, then rrtmgp uses this instead of co2vmr? Why not just changing the value of co2vmr?

Reply:
That's a good question. Short answer is "that's what EAM does". I think this is to support overriding the value of co2vmr *just* for the rad calculations. I don't really see what the point of this is, so I think I'll just remove this...

Reply:
Removed.